terraform/tfschema/types_terraform.go

```diff
@@ -1346,38 +1364,56 @@ func GenSchemaMetaProvisionTokenV2() map[string]*accessors.SchemaMeta {
 					IsTime:     false,
 					IsDuration: false,
 				},
-
+				// Allow is a list of TokenRules, nodes using this token must match one
+				// allow rule to use this token.
 				"allow": {
-					Name:       "allow",
```

Comment:
This is the line which fixes #427

Reply:
@pschisa @programmerq Can you confirm that this fixes the issue?

Reply:
I just built this from the branch and confirmed that it this change fixes the issue!